woocommerce-connect-client.php

```diff
@@ -632,6 +632,7 @@ public function admin_enqueue_scripts() {
 			wp_register_script( 'wc_connect_admin', $wc_connect_base_url . 'woocommerce-connect-client.js', array(), false, true );
 
 			require_once( plugin_basename( 'i18n/strings.php' ) );
+			// TODO: Where is $i18nStrings defined?
```

Comment:
Here: https://github.com/Automattic/woocommerce-connect-client/blob/master/woocommerce-connect-client.php#L634

I don't know why PHPStorm chokes on it, though.